### DISCUSSION TITLE
Geographical scope of Chirpity

### DISCUSSION THREAD

**Discussion by paulcoiffard** [Mattk70/Chirpity-Electron#161] [Q&A] [answered]
Hi

What species has Chirpity been trained on? Only British species or all European species? 
I'd like to know if it would it be biased to use it on nocmig recordings made in France, for example? With species that are quite rare in the UK but locally well represented in France, such as the Gull-billed tern, Great Spotted Cuckoo, Western Swamphen etc.

Thanks for your amazing work, that's a wonderful tool.

Cheers,

Paul

**Comment by Mattk70** [comment, marked as answer]
Hi @paulcoiffard, if you check Help > What species are detected? Chirpity will show you the species known to the currently selected model (a combination of excluded and included according to the list you are using) 

I'm looking for species requests in this discussion: https://github.com/Mattk70/Chirpity-Electron/discussions/144, feel free to post any European species you would like me to add to the Nocmig model.

**Reply by paulcoiffard** [reply]
Hi @Mattk70, thanks a lot for your answer, sorry I've missed that. I'll answer in #144 in a few minutes :)
